### PR TITLE
Bring back server pinger

### LINKED ISSUE #417: Re-integrate server-pinger Scala/SBT project
**Rules: http://work.scalawilliam.com/rules/**

--

As there's no benefit to having it as a separate library. Similar to #408 but a little simpler due to no subprojects to manage.

- Git/SBT project: https://github.com/ActionFPS/server-pinger
  - Take SP repo and move its root contents into a sub-directory `server-pinger`.
  - Merge new SP repo into ActionFPS repo (this repo) using this technique https://stackoverflow.com/a/10548919 (but put into `server-pinger` directory).
  - Integrate SP's `build.sbt` into ActionFPS `actionfps.sbt`, excluding all the publishing features. No `built.sbt` should remain afterwards. Put dependencies into `Dependencies.scala`.
  - Remove reference to server-pinger in `Dependencies.scala` of the SBT build, instead referring to the local project
  - Make sure README.md is up to date.
  - I may have missed something else that needs to be removed, so scope may creep a little
- Remove references from Docs: https://github.com/ActionFPS/ActionFPS-Book/blob/master/portal-development-guide.md

I'll pay $30 for this ([ScalaWilliam Work](http://work.scalawilliam.com/integrate-project-pinger-back/)).
Please post here "**I'm taking on this**" to lock the task to yourself.

### PR DESCRIPTION
**Pull request by omainegra**
This closes #417 and it's related to ActionFPS/ActionFPS-Book#10

### CODE REVIEW

**Comment by ScalaWilliam** [comment]
Quite busy with dayjob, will take a look soon..

File: actionfps.sbt [hunk 1]
```diff
@@ -446,6 +447,16 @@ lazy val gameLogParser =
       libraryDependencies += scalatest % Test
     )
 
+lazy val serverPinger =
+  Project(id = "server-pinger", base = file("server-pinger"))
+    .settings(
+      libraryDependencies ++= Seq(jodaTime, jodaConvert, commonsIO, playJson, akkaActor),
+      libraryDependencies += scalatest % Test, {
+        lazy val FunTest = config("fun") extend (Test)
```
Comment (ScalaWilliam) [inline actionfps.sbt]: This definition can live outside of this code block

File: actionfps.sbt [hunk 2]
```diff
@@ -446,6 +447,16 @@ lazy val gameLogParser =
...
+        configs(FunTest)
```
Comment (ScalaWilliam) [inline actionfps.sbt]: This can be applied as .configs on serverPinger. See lines 108-109

File: actionfps.sbt [hunk 3]
```diff
@@ -446,6 +447,16 @@ lazy val gameLogParser =
       libraryDependencies += scalatest % Test
     )
 
+lazy val serverPinger =
+  Project(id = "server-pinger", base = file("server-pinger"))
```
Comment (ScalaWilliam) [inline actionfps.sbt]: Would you scalafmt this file? :-)

File: actionfps.sbt (hunk 1 above)
Comment (omainegra) [reply actionfps.sbt]: Done

File: actionfps.sbt (hunk 2 above)
Comment (omainegra) [reply actionfps.sbt]: Done

File: actionfps.sbt (hunk 3 above)
Comment (omainegra) [reply actionfps.sbt]: Done

**Pull request merged**